xrdb: add page (#6495)
tldr-pages/tldr: xrdb.md

# xrdb

> X window server's resource database utility for Unix-like systems.

> > More information: https://www.x.org/releases/X11R7.7/doc/man/man1/xrdb.1.xhtml

Reviewer: We use this formatting here:
```suggestion
> More information: <https://www.x.org/releases/X11R7.7/doc/man/man1/xrdb.1.xhtml>.
```

> - Run the utility in interactive mode:

Reviewer 2: ```suggestion
- Start xrdb in interactive mode:
```

>  `xrdb`

Reviewer 2: There is still a space here which should be. Please apply these suggestions from here rather than just editing the page yourself.

```suggestion
`xrdb`
```

> - Load values (eg. style rules) from a resource file:

Reviewer 3: ```suggestion
- Load values (e.g. style rules) from a resource file:
```

>  `xrdb -load ~/.Xresources`

Reviewer 2: Is this changeable?

If so then please apply this

```suggestion
 `xrdb -load {{~/.Xresources}}`
```

Reviewer 2: ```suggestion
`xrdb -load ~/.Xresources`
```

- Query the resource database and print currently set values:

>  `xrdb -query`

Reviewer 2: ```suggestion
`xrdb -query`
```

# xrdb

> X window server's resource database utility for Unix-like systems.

Reviewer 2: ```suggestion
> X window server's resource database utility for Unix-like systems.
```

> 

Reviewer 2: Extra line is unrequired

> More information: https://www.x.org/releases/X11R7.7/doc/man/man1/xrdb.1.xhtml

Reviewer 2: ```suggestion
> More information: <https://www.x.org/releases/X11R7.7/doc/man/man1/xrdb.1.xhtml>.
```

- Run the utility in interactive mode:

>   `xrdb`

Reviewer 2: ```suggestion
`xrdb`
```

> - Load values (eg. style rules) from a resource file

Reviewer 2: ```suggestion
- Load values (eg. style rules) from a resource file:
```

>   `xrdb -load ~/.Xresources`

Reviewer 2: ```suggestion
`xrdb -load ~/.Xresources`
```

- Query the resource database and print currently set values:

>   `xrdb -query`

Reviewer 2: ```suggestion
`xrdb -query`
```
